### ISSUE TITLE
Cannot export JLS v2 to JLS v1

### ISSUE THREAD

**Issue by mliberty1** [jetperch/pyjoulescope_ui#142]
Version: 0.9.11 [in progress]
Platform: All

**Description**

If you load a JLS v2 file, then select a region with dual markers, right click, export, then the software throws an exception:

```
ERROR:2022-02-17 16:25:58,838:range_tool.py:205:joulescope_ui.range_tool:range tool run exception
Traceback (most recent call last):
  File "C:\repos\Jetperch\pyjoulescope_ui\joulescope_ui\range_tool.py", line 201, in _thread_run
    rv = self._range_tool_obj.run(self)
  File "C:\repos\Jetperch\pyjoulescope_ui\joulescope_ui\exporter.py", line 59, in run
    rv = fn(data)
  File "C:\repos\Jetperch\pyjoulescope_ui\joulescope_ui\exporter.py", line 116, in _export_jls
    calibration=data.calibration.data)
AttributeError: 'NoneType' object has no attribute 'data'
WARNING:2022-02-17 16:25:58,867:main.py:1795:joulescope_ui.main:Export data: ERROR
```

However, the problem is actually more complicated than just skipping writing the calibration data.  The content of JLS v2 files varies, while the content of JLS v1 is fixed.  JLS v2 files cannot in general be saved as JLS v1.  The export function currently only writes to JLS v2.

**Expected Behavior**
JLS v2 files should export to JLS v2.

**Comment by mliberty1** [comment]
The other JLS v2 range tool issues were fixed in #131.  We will likely defer fixing this issue.  It will be fixed "automatically" when the the UI transitions to JLS v2 by default.

**Comment by mliberty1** [comment]
Starting with 1.0.0, the UI writes JLS v2 files only.  This issue no longer applies.
